src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,3 +1,32 @@
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <>
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      {' Posted by  '}
```

Comment:
```suggestion
      Posted by
```

you may use a plain text in jsx